internal/kuberesource/sets.go

```diff
@@ -151,6 +152,37 @@ func GetDEnts() ([]any, error) {
 	return []any{tester}, nil
 }
 
+// GenpolicyRegressionTests returns deployments for regression testing genpolicy.
+func GenpolicyRegressionTests() map[string]*applyappsv1.DeploymentApplyConfiguration {
+	out := make(map[string]*applyappsv1.DeploymentApplyConfiguration)
+
+	badLayer := "bad-layer"
+	out[badLayer] = Deployment(badLayer, "").
+		WithSpec(DeploymentSpec().
+			WithReplicas(1).
+			WithSelector(LabelSelector().
+				WithMatchLabels(map[string]string{"app.kubernetes.io/name": badLayer}),
+			).
+			WithTemplate(PodTemplateSpec().
+				WithLabels(map[string]string{"app.kubernetes.io/name": badLayer}).
+				WithSpec(PodSpec().
+					WithRuntimeClassName(runtimeHandler).
+					WithContainers(
+						Container().
+							WithName(badLayer).
+							WithImage("ghcr.io/edgelesssys/contrast/genpolicy-bad-layer:1").
```

Comment:
Where is the definition of this image?

Reply:
This is a hand-crafted image, adding a known-bad layer to `busybox:3.16.1`. That one is pretty hard to reproduce.

On the other hand, I think this issue should be reproducible with a simple non-ASCII link name. In that case, I could write a Dockerfile for it. But where should we put that? Do we have a definition for the getdents image?

Reply:
> But where should we put that? Do we have a definition for the getdents image?

I'm fine with putting it into e2e/<testname>/Containerfile or into tools/e2e-containers. I don't think we have the source of the getdents image, but I'd be really happy to fix it.

Reply:
I'm using an `httpd` image from dockerhub for now, don't think we need the source for that.